lib/webapi/webhandler.go

```diff
@@ -307,8 +309,8 @@ func (h *WebHandler) siteHandler(w http.ResponseWriter, r *http.Request, p httpr
 	}
 
 	// if the app defines custom install step and the site hasn't completed it yet,
-	// always redirect the site to it
-	http.Redirect(w, r, "/web/installer/site/"+siteDomain+"/complete/", http.StatusFound)
+	// always redirect the site back to installer
+	http.Redirect(w, r, "/web/installer/site/"+siteDomain, http.StatusFound)
```

Comment:
This means the user will see "installation complete" page again and will have to click "continue", right? Would it be complicated to fix this session storage issue you told me about and redirect directly to bandwagon instead?

Reply:
This approach is simpler as we do not have much control over customized bandwagon step